kratos/python/add_cfd_variables_to_python.h

```diff
@@ -4,39 +4,25 @@
 //   _|\_\_|  \__,_|\__|\___/ ____/
 //                   Multi-Physics
 //
-//  License:		 BSD License
-//					 Kratos default license: kratos/license.txt
+//  License:         BSD License
+//                   Kratos default license: kratos/license.txt
 //
-//  Main authors:    Pooyan Dadvand
-//                   Riccardo Rossi
```

Comment:
Same comment. I'd leave the original author order.

Reply:
(Okay, for me the copy pasting was very practical)